docs/pages/admin-guide.mdx

```diff
@@ -677,17 +633,15 @@ server IP with the URL of the proxy server. In the example below, we've
 replaced the auth server IP with the proxy web endpoint `teleport-proxy.example.com:3080`.
 
 ```bash
-$ sudo tctl nodes add
+tctl tokens add --type=node | grep -oP '(?<=token:\s).*' > token.file
 
-The invite token: n92bb958ce97f761da978d08c35c54a5c
-Run this on the new node to join the cluster:
-teleport start --roles=node --token=n92bb958ce97f761da978d08c35c54a5c --auth-server=teleport-proxy.example.com:3080
+# This will save the token to a file.
+# Run this on the new node to join the cluster:
+sudo teleport start --roles=node --token=/path/to/token.file --auth-server=teleport-proxy.example.com:3080
 ```
 
-Using the ports in the default configuration, the node needs to be able to talk to ports 3080
-and 3024 on the proxy. Port 3080 is used to initially fetch the credentials (SSH and TLS certificates)
-and for discovery (where is the reverse tunnel running, in this case 3024). Port 3024 is used to
-establish a connection to the auth server through the proxy.
+Using the ports in the default configuration, the node needs to be able to talk to ports `3080` and `3024` on the proxy. Port `3080` is used to initially fetch the credentials (SSH and TLS certificates)
```

Comment:
We should add a note that if using the quickstart Acme will use `:443` vs `3080`